code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/weapon/reagent_containers/food/drinks/cans
-	var canopened = 0
-	var is_glass = 0
+	var/canopened = 0
+	var/is_glass = 0
+	var/is_plastic = 0
```

Comment:
If you ever end up with two mutually exclusive flags like this, it's usually a good sign that you shouldn't still be using flags. In this case, you probably want to replace both of these with a single number, using `#define`d values, representing the can's material.